<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SegmentationClass</class>
 <widget class="QMainWindow" name="SegmentationClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Segmentation</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="timeLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Time:</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>163</width>
      <height>381</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QListWidget" name="imageList"/>
     </item>
     <item>
      <widget class="QPushButton" name="objButton">
       <property name="text">
        <string>set object color</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bkgButton">
       <property name="text">
        <string>set background color</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="seedButton">
       <property name="text">
        <string>set seed path</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="gcRadio">
       <property name="text">
        <string>graph cut</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="gbRadio">
       <property name="text">
        <string>grab cut</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="mgcRadio">
       <property name="text">
        <string>multi-level graph cut</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="gddRadio">
       <property name="text">
        <string>geodesic distance depth</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rgbdRadio">
       <property name="text">
        <string>RGBD grapg cut</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="ggRadio">
       <property name="text">
        <string>GG</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="hggRadio">
       <property name="text">
        <string>HGG</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="saveButton">
         <property name="text">
          <string>save</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>511</width>
      <height>361</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="RGBImage">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="depthImage">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="seedImage">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="resultImage">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>697</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpen">
    <property name="title">
     <string>open</string>
    </property>
    <addaction name="OpenRGB"/>
    <addaction name="OpenDepth"/>
   </widget>
   <addaction name="menuOpen"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="OpenRGB">
   <property name="text">
    <string>open RGB</string>
   </property>
  </action>
  <action name="OpenDepth">
   <property name="text">
    <string>open Depth</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="segmentation.qrc"/>
 </resources>
 <connections/>
</ui>
